test/Tests/polyglot-sources/enso-test-java-helpers/src/main/java/org/enso/table_test_helpers/PolyglotHelpers.java

```diff
@@ -0,0 +1,16 @@
+package org.enso.table_test_helpers;
+
+import java.math.BigInteger;
+
+public class PolyglotHelpers {
+  /* Creates a big integer that is larger than long can fit. */
+  public static BigInteger createBigBigIntegerComingFromJava() {
+    return BigInteger.TWO.pow(70);
```

Comment:
is it worth making this no something that can be stored in a double exactly?
```suggestion
    return BigInteger.TWO.pow(70).minus(1);
```

Reply:
Actually, `2^70` is _not_ exactly representable.

I was surprised, but that seems to be the case:
```
> 2^70
>>> 1180591620717411303424
> (2^70)+0.0
>>> 1.1805916207174113E21
> (2^70)+0.0 . truncate
>>> 1180591620717411300000
```

At least the roundtrip back to integer is inexact - that is because the conversion from `double` to `BigDecimal` happens at base 10 and not 2.

Reply:
(But I still have changed the test in question)